docs/src/features.md

```diff
@@ -7,26 +7,80 @@
+Therefore, for this simple example we can determine the number of constraints and variables in the optimization problem:
+
+- *Number of variables*: 42 since we have six connections among assets (i.e., 6 flows x 6 hours = 36 variables) and one storage asset (i.e., 6 storage level x 6 h = 6 variables)
+- *Number of constraints*: 72, where:
+
+  - 24 from the maximum output limit of the assets that produce, convert, or discharge energy (i.e., `H2`, `wind`, `ccgt`, and `phs`) for each hour (i.e., 4 assets x 6 h = 24 constraints)
+  - 6 from the maximum input limit of the storage or charging limit for the `phs`
+  - 6 from the maximum storage level limit for the `phs`
+  - 12 from the import and export limits for the transmission line between the `balance` hub and the `demand`
+  - 24 from the energy balance on the consumer, hub, conversion, and storage assets (i.e., `demand`, `balance`, `ccgt`, and `phs`) for each hour (i.e., 4 assets x 6 h = 24 constraints)
+
+Depending on the input data and the level of detail you want to model, hourly resolution in all the variables might not be necessary. *TulipaEnergyModel.jl* has the possibility to have different time resolutions for each asset and flow to simplify the optimization problem and approximate hourly representation. This feature is particularly useful for large-scale energy systems that involve different sectors, as detailed granularity is not always necessary due to the unique temporal dynamics of each sector. For instance, we can use hourly resolution for the electricity sector and six-hour resolution for the hydrogen sector. We can couple multiple sectors, each with its temporal resolution.
+
+Let's explore the flexibility in the time resolution with the following examples. The following table shows the user input data for the asset time resolution definition. Please note that the values presented in this example are just for illustrative purposes and do not represent a realistic case.
+
+```@example print-partitions
+using DataFrames # hide
+using CSV # hide
+input_asset_file = "../../test/inputs/Variable Resolution/assets-partitions.csv" # hide
+assets = CSV.read(input_asset_file, DataFrame, header = 2) # hide
+assets = assets[assets.asset .!= "wind", :] # hide
+```
```

Comment:
I'm not familiar with `# hide`. So just pointing out that you have this in every line. But most probably that is intentional.

Reply:
The `#hide` is to avoid showing this code in the documentation. Here, I only want to call the file and print it in a table. If you clone the branch and create the documentation locally, you will see how it renders. This is a trick from Abel 😉